articles/articles/article.php

```diff
@@ -1,176 +1,321 @@
 <?php
 /**
- * @package	API
- * @version 1.5
- * @author 	Brian Edgerton
- * @link 	http://www.edgewebworks.com
- * @copyright Copyright (C) 2011 Edge Web Works, LLC. All rights reserved.
- * @license http://www.gnu.org/copyleft/gpl.html GNU/GPL, see LICENSE.php
-*/
-
+ * @package    Com_Api
+ * @copyright  Copyright (C) 2009-2014 Techjoomla, Tekdi Technologies Pvt. Ltd. All rights reserved.
+ * @license    GNU GPLv2 <http://www.gnu.org/licenses/old-licenses/gpl-2.0.html>
+ * @link       http://www.techjoomla.com
+ */
 defined('_JEXEC') or die( 'Restricted access' );
+require_once JPATH_SITE . '/components/com_content/models/articles.php';
+require_once JPATH_SITE . '/components/com_content/models/article.php';
 
-jimport('joomla.plugin.plugin');
-jimport('joomla.html.html');
-jimport('joomla.user.helper');
-jimport('joomla.application.component.model');
-jimport( 'joomla.application.component.model' );
-jimport( 'joomla.database.table.user' );
-jimport( 'joomla.libraries' );
-jimport( 'joomla.media.foundry' );
-
-use Joomla\Registry\Registry;
-
-require_once JPATH_SITE .'/components/com_content/models/articles.php';
-require_once JPATH_SITE .'/components/com_content/models/article.php';
-require_once JPATH_SITE .'/libraries/joomla/document/html/html.php';
-require_once JPATH_SITE .'/libraries/cms/plugin/plugin.php';
-require_once JPATH_SITE .'/libraries/cms/plugin/helper.php';
-
-class ArticlesApiResourceArticle extends ApiResource
+/**
+ * Articles Resource
+ *
+ * @since  3.5
+ */
+class ContentApiResourceArticles extends ApiResource
 {
+	/**
+	 * get Method to get all artcle data
+	 *
+	 * @return  json
+	 *
+	 * @since  3.5
+	 */
 	public function get()
 	{
-		//init variable
+		$this->plugin->setResponse($this->getArticles());
+	}
+
+	/**
+	 * delete Method to delete artcle
+	 *
+	 * @return  json
+	 *
+	 * @since  3.5
+	 */
+	public function delete()
+	{
+		$this->plugin->setResponse('in delete');
+	}
+
+	/**
+	 * getArticles Method to getArticles data
+	 *
+	 * @return  array
+	 *
+	 * @since  3.5
+	 */
+	public function getArticles()
+	{
 		$app = JFactory::getApplication();
-		
-		$result = new stdClass();
 		$items = array();
-		$article_id	= $app->input->get('id', 0, 'INT');
-		$catid	= $app->input->get('category_id', 0, 'INT');
-		
-		//featured - hide,only,show
+		$article_id = $app->input->get('id', 0, 'INT');
+		$catid = $app->input->get('category_id', 0, 'INT');
+
+		// Featured - hide,only,show
 		$featured	= $app->input->get('featured', '', 'STRING');
 		$auther_id	= $app->input->get('auther_id', 0, 'INT');
```

Comment:
@vijaykhollam change auther_id to created_by everywhere please. Also the type for $featured should be int not string.